### PR TITLE
Joelene

### PR DESCRIPTION
**Pull request by joelenelatief**
Added Google sign in button and login screen component 

image

### CODE REVIEW

File: app/javascript/components/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { TextField, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@material-ui/core/';
+
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this._handleLogin = this._handleLogin.bind(this);
+  }
+
+  componentDidMount () {
```
Comment (adowski) [inline app/javascript/components/Login.jsx]: Can delete commented-out lines!

File: app/javascript/components/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  render() {
+    return (
+        <div className="login-container">
+          <div className="login">
+          <div className="login-box">
+            <h2 className="title"> Welcome, please sign in:</h2>
+            <img src={require('./unloop_logo_teal.png')} width={300}  mode='fill'/>
+            <div className="sign-in-button" >
+              {/* <div class="g-signin2" data-width="250" data-height="50" data-longtitle="true" onClick={this._handleLogin}></div> */}
```
Comment (adowski) [inline app/javascript/components/Login.jsx]: Delete commented-out lines

File: app/javascript/components/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+              <img src={require("./btn_google_signin_light_normal_web@2x.png")} width ="200" onClick={this._handleLogin}></img>
+            </div>
+          </div>
+        </div>
+      </div>
+      
+      // <div>
```
Comment (adowski) [inline app/javascript/components/Login.jsx]: Same as above

**Comment by joelenelatief** [comment]
Updated styling to align with highfi and put in official google sign in button

image

**Comment by joelenelatief** [comment]
i removed alison's face :( 
image

**Pull request merged**